packages/exec/__tests__/scripts/print-args-cmd.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
the naming of these script files is a little bit weird, but it's related to the conventions set forth by the existing test case scripts

Reply:
i just copied one of the existing cmd files (but it had spaces in the path, and i didnt want that)

this script basically just echoes the args, e.g.

```
arg[0]: "hello"
arg[1]: "world"
```